test/e2e/pkg/testnet.go

```diff
@@ -81,8 +81,8 @@ type Testnet struct {
 	Dir  string
 
 	IP               *net.IPNet
-	Validators       map[*Node]int64
-	ValidatorUpdates map[int64]map[*Node]int64
+	Validators       map[string]int64
```

Comment:
Since the type of `Validators` here is now identical to in `pkg/manifest.go` we don't even need to keep it here.

Namely, you could only rename `ValidatorsMap` to `Validators` in `Manifest` and remove this field from here.

Reply:
See https://github.com/cometbft/cometbft/pull/3964 for more details.

Reply:
Addressed

Reply:
We can rename it back to `Validators`, see https://github.com/cometbft/cometbft/pull/4166